src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -69,33 +78,34 @@ protected void checkChimericInstances(QAReport report, String clsName) throws Ex
             if (isEscaped(rle)) {
                 continue;
             }
-            if (QACheckerHelper.isChimeric(rle)) {
-                if (!hasMultipleSpecies(rle))
-                    report.addLine(rle.getDBID() + "",
-                                   rle.getDisplayName(),
-                                   rle.getSchemClass().getName(),
-                                   "Chimeric but one or null species",
-                                   QACheckerHelper.getLastModificationAuthor(rle));
-                if (isNotUsedForInference(rle))
-                    report.addLine(rle.getDBID() + "",
-                                   rle.getDisplayName(),
-                                   rle.getSchemClass().getName(),
-                                   "Chimeric but not used for inference",
-                                   QACheckerHelper.getLastModificationAuthor(rle));
-            }
-            else {
-                if (hasMultipleSpecies(rle))
-                    report.addLine(rle.getDBID() + "",
-                                   rle.getDisplayName(),
-                                   rle.getSchemClass().getName(),
-                                   "Not chimeric but with multiple species",
-                                   QACheckerHelper.getLastModificationAuthor(rle));
-                if (isParticipantChimeric(rle)) {
-                    report.addLine(rle.getDBID() + "",
-                                   rle.getDisplayName(),
-                                   rle.getSchemClass().getName(),
-                                   "Not chimeric but with chimeric participant",
-                                   QACheckerHelper.getLastModificationAuthor(rle));
+            if (!skipList.containsInstanceDbId(rle)) {
```

Comment:
I'm just wondering if this method would be better named `containsInstance` since it takes a `GKInstance` object or if it would be better to rework it to take in a `long` value and then say `containsInstanceDbId(rle.dbId())`.

Reply:
Done